### DISCUSSION TITLE
Dividing “Initial Draft Variant” into “Prelude Draft” and “First Ten Cards Draft”

### DISCUSSION THREAD

**Discussion by GAS90000** [bafolts/terraforming-mars#5873] [Ideas]
Is it possible to divide “Initial Draft Variant” into “Prelude Draft” and “First Ten Cards Draft”? We usually draft the first ten cards but don’t draft Prelude cards.

**Comment by kberg** [comment]
<img width="888" height="388" alt="image" src="https://github.com/user-attachments/assets/2ce18d9f-b81e-43af-91e6-548c6b82ce90" />
Initial draft is now possible without the prelude draft
